mythril/laser/smt/bitvec_helper.py

```diff
@@ -32,6 +48,11 @@ def If(a: Union[Bool, bool], b: Union[BitVec, int], c: Union[BitVec, int]) -> Bi
     """
     if not isinstance(a, Bool):
         a = Bool(z3.BoolVal(a))
+
+    if isinstance(b, BaseArray) and isinstance(c, BaseArray):
+        array = z3.If(a.raw, b.raw, c.raw)
+        return z3_array_converter(array)
```

Comment:
Is z3_array_converter going to be used anywhere else? 

Otherwise it might be better to not separate this in a function / make z3_array_converter a private function in `bitvec_helper.py` WDYT?

Reply:
The file name was bitvec_helper.py so I didn’t want to place it here as the function was array specific. But we can make it a private function in this file.